emanuilnikolov/igdb-api-bundle/1.0/config/packages/en_igdb_api.yaml

```
@@ -0,0 +1,4 @@
+# Get your credentials by signing up here: https://api.igdb.com/admin
+en_igdb_api:
+    base_url: "YOUR_BASE_URL"
```

You can use the env var like this :
```suggestion
    base_url: "'%env(YOUR_BASE_URL)%'"
```